pkgs/build-support/fetchpijul/default.nix

```diff
@@ -0,0 +1,24 @@
+{ stdenvNoCC, pijul, lib }:
+
+lib.makeOverridable (
+{ url
+, sha256 ? ""
+, change ? null
+, state ? null
+, channel ? "main"
+, name ? "fetchpijul"
+}:
+if change != null && state != null then
+  throw "Only one of change or state can be set"
+else
+  stdenvNoCC.mkDerivation {
+    inherit name;
+    builder = ./builder.sh;
```

Comment:
We don't necessarily need to use builder and shouldn't probably add new usages.

Reply:
I'm sorry, I don't understand what you mean by this comment

Reply:
He means that you can just use the default stdenv stages/`buildCommand`, no need to explicitly define a custom builder.

Reply:
I see, so I suspected correctly. Is there any benefit for doing that? I wanted to stay as consistent with the fetchers we already have as I can, but maybe that's not the consensus.

Reply:
The trend is to stick with standard stages rather than using custom builders (e.g. #237856, #238764) which ultimately reduces complexity/maintenance cost. Please remove the custom builder.